openstack/keystone/templates/deployment-2faproxy.yaml

```diff
@@ -35,6 +35,8 @@ spec:
             - -exclude-domain={{ $domain }}
             {{- end}}
           env:
+            - value: OS_AUTH_URL
+              name: "http://$(KEYSTONE_SERVICE_HOST):$(KEYSTONE_SERVICE_PORT)/v3"
```

Comment:
could you please explain, why we have http here and not http**s**? My local `OS_AUTH_URL` starts with https.

Reply:
that's the backend connection from the 2faproxy to the keystone-api, its behind the ingress which does the tls termination:

```
[client] --https--> [nginx-ingress ] --http--> [2faproxy] --http--> [keystone]
```